lib/forest_liana/bootstraper.rb

```diff
@@ -174,6 +178,12 @@ def create_apimap
       end
     end
 
+    def create_apimap_json
+      File.open(File.join(Rails.root, 'forestadmin-schema.json'), 'w') do |f|
+        f.puts JSON.pretty_generate(ForestLiana.apimap.as_json)
+      end
```

Comment:
We should print a clear message if:

    the file is invalid
    the file is empty
    the file is not present

Reply:
Not in this PR we are just creating the file